CRM/Chreports/Upgrader.php

```diff
@@ -1121,4 +1121,49 @@ public function upgrade_102009() {
     return TRUE;
   }
 
+  public function upgrade_102010() {
+    $this->ctx->log->info('Reporting v1.2.3 (#010): Add "Batch Name" Filter to Report Filter list & Template Columns');
+    $non_migrated_templates = E::getNonMigratedReportTemplates();
+    // Fetch all saved reports
+    $reportInstances = civicrm_api3('ReportInstance', 'get', [
+      'sequential' => 1,
+      'created_id' => ['IS NOT NULL' => 1],
```

Comment:
We don't need to retrieve ALL reports, only the ones we've migrated to our code base. 

While this is unlikely to have much impact on performance on < 500 records, it's important to get into a habit of optimizing.
 
```suggestion
      'created_id' => ['IS NOT NULL' => 1],
      'report_id' => ['LIKE' => "chreports/%"],
```